src/Modules/Visualization/ShowField.cc

```diff
@@ -926,6 +926,13 @@ void GeometryBuilder::renderEdges(
   ColorScheme colorScheme;
   ColorRGB edge_colors[2];
 
+  ColorMapHandle realColorMap = nullptr;
+  if(colorMap) realColorMap = colorMap.get();
+  if(!realColorMap) realColorMap = StandardColorMapFactory::create();
+
+  auto coordinateMap = StandardColorMapFactory::create("Grayscale", 256, 0, false,
+    realColorMap->getColorMapRescaleScale(), realColorMap->getColorMapRescaleShift());
```

Comment:
This is duplicated in the face code! Time to make a function/class to handle new colormap creation.